src/track/info.ts

```diff
@@ -0,0 +1,245 @@
+import {
```

Comment:
in the future, we might want to move this logic to another package (i.e. one that will be focused primarily on tracking and analytics)